src/test/java/org/repodriller/scm/git/GitRemoteRepositoryTest.java

```diff
@@ -53,37 +57,42 @@ public void shouldGetInfoFromARepo() {
 		SCMRepository repo = git1.info();
 		Assert.assertEquals("c79c45449201edb2895f48144a3b29cdce7c6f47", repo.getFirstCommit());
 	}
-	
+
 	@Test
 	public void shouldGetSameOriginURL() {
 		SCMRepository repo = git1.info();
 		String origin = repo.getOrigin();
 		Assert.assertEquals(url, origin);
 	}
-	
+
 	@Test
 	public void shouldInitWithGivenTempDir() {
-		String expectedRepoTempDirectory = new File(REMOTE_GIT_TEMP_DIR + File.separator + "repodriller").getAbsolutePath();
-		Assert.assertEquals(expectedRepoTempDirectory, git2.info().getPath());
-		
-		File bareRepositoryRefDir = new File(expectedRepoTempDirectory + File.separator + "refs");
-		Assert.assertTrue("A bare repository should have refs directory.", bareRepositoryRefDir.exists());
+		Path expectedStart = Paths.get(REMOTE_GIT_TEMP_DIR).toAbsolutePath();
+		Assert.assertTrue("Directory " + REMOTE_GIT_TEMP_DIR + " not honored. Path is " + git2.info().getPath(),
+				Paths.get(git2.info().getPath()).startsWith(expectedStart));
+
+		File bareRepositoryRefDir = new File(git2.info().getPath() + File.separator + "refs");
+		Assert.assertTrue("A bare repository should have a refs directory",
+				bareRepositoryRefDir.exists());
 	}
-	
+
 	/**
 	 * Doesn't work in every machine/filesystem.
 	 * Mock to avoid this issue and make test independent of internet connection?
 	 */
+	/* TODO This works on my Linux box. Can someone test on Windows? */
 //	@AfterClass
```

Comment:
is this really supposed to be commented?

Reply:
Ah, so you mean you tried on linux, and you wanted to know if it works on windows! I'd say, if it works on our CI, then we should have it! :)

Reply:
OK. I'll try uncommenting it and we'll see if Travis blows up.

Reply:
See a03dbd8.